ui/src/pages/overview/index.tsx

```diff
@@ -0,0 +1,10 @@
+import { useGitVersion } from './useGitVersion';
+
+export function OverviewComponent() {
+	const GitVersion = useGitVersion();
+	return (
+		<>
+			<GitVersion />
+		</>
```

Comment:
Nit: can this be done without wrapping a single component in a fragment?

```suggestion
		<GitVersion />
```

Reply:
Yeah, but it's changing in the very next PR - the idea would be to add multiple elements there for a few different "Overview" elements, and having the fragment would help with multiple PRs that don't have conflicts.